src/pages/VaccineModulePage.js

```diff
@@ -19,6 +19,8 @@ import globalStyles, {
   HAZARD_RED,
 } from '../globalStyles';
 
+const MILLISECONDS_IN_DAY = 24 * 60 * 60 * 10000;
```

Comment:
`1000` not `10000` - also I just found this constant in `database/utilities` also

Reply:
fixed in last commit